### ISSUE TITLE
Use reference for content-type identifier

### ISSUE THREAD

**Issue by TFohrer** [kaliop-uk/ezmigrationbundle#157]
Hi !

First of all thanks for your great bundle. Amazing work.

I'm trying to use a reference as a content_type identifier:

```
- 
    type: content_type
    mode: create
    content_type_group: 'Pages'
    identifier: reference:content_type__pages__home
    ...
    ...
```

Storing the references in a seperate file and loading them via: 

```
-
    type: reference
    mode: load
    file: 'src/My/Amazing/Bundle/References/references_content_types.yml'
    overwrite: true
```

And the references_content_types.yml would be something like this: 

```
ref_content_type__pages__home: scaya_pages_home
```

Am I doing something wrong or can't you use references the way I'm trying to ? 

Thanks for your help.

**Comment by gggeek** [comment]
Hello. Sorry for taking so long for answering this.

I just checked the code, and indeed ref. resolution for ContentType Identifiers is not supported...

...but it seems a legit request. Looking into implementing it :-)

**Comment by gggeek** [comment]
Implemented in 4.6.0
